microproxy/protocol/http2.py

```diff
@@ -166,26 +183,28 @@ def handle_update_settings(self, event):
         self.on_settings(event.changed_settings)
 
     def handle_window_updates(self, event):
-        self.on_window_updates(event.stream_id, event.delta)
+        self.on_window_updates(event.stream_id, event.delta, event)
```

Comment:
If all callback send event as parameter, then I think there is no need to send the field of the event.
It's duplicate!